sympy/functions/combinatorial/tests/test_comb_factorials.py

```diff
@@ -691,6 +695,7 @@ def test_multinomial():
     assert multinomial(z) == 1
     assert multinomial(z, n, k) == binomial(k + n, k)
     assert unchanged(multinomial, z, z, n, k, k) is False
+    assert multinomial(*[x]*5) == factorial(5*x)/factorial(x)**5
```

Comment:
`rf(x + 1, 4*x)/factorial(x)**4`...maybe that case should be recast in this way?
```python
>>> x = 5
>>> factorial(5*x)/factorial(x)**5
623360743125120
>>> rf(x+1,4*x)/factorial(x)**4
623360743125120
```

Reply:
do you mean to change the test case, or the way it returns in eval ?

Reply:
Perhaps it's fine. `factorial` is pretty smart about evaluation and replacing `factorial` with `rf` might counteract that.